pandas/core/reshape/concat.py

```diff
@@ -44,6 +45,8 @@
 from pandas.core.internals import concatenate_managers
 
 if TYPE_CHECKING:
+    from pandas._typing import Axis
```

Comment:
imports from pandas._typing are guarded, so no need to put inside TYPE_CHECKING block.